### PR TITLE
Update site pages

### PR DESCRIPTION
**Pull request by gcapurso**
Accordingly to the last design: https://binnenland.atlassian.net/browse/OP-405 >> https://miro.com/app/board/o9J_l9uYRvM=/?moveToWidget=3074457361016755397&cot=14

### CODE REVIEW

File: app/templates/administrative-units/administrative-unit/sites/index.hbs [hunk 1]
```diff
@@ -30,7 +31,8 @@
           {{site.address.fullAddress}}
         </AuLink>
       </td>
-      <td>{{if (eq site.id this.model.primarySite.id) "Ja" "Nee"}}</td>
+      <td>{{site.siteTypes.label}}</td>
```
Comment (Windvis) [inline app/templates/administrative-units/administrative-unit/sites/index.hbs:34]: I think this should be `siteType`, right?

File: app/templates/administrative-units/administrative-unit/sites/index.hbs [hunk 2]
```diff
@@ -30,7 +31,8 @@
...
+      <td>{{if (eq site.id this.model.primarySite.id) "Ja" "NeeN"}}</td>
```
Comment (Windvis) [inline app/templates/administrative-units/administrative-unit/sites/index.hbs:35]: Typo, should be "Nee" I think?

**Review by Windvis — CHANGES_REQUESTED** [review]

File: app/templates/administrative-units/administrative-unit/sites/index.hbs (hunk 2 above)
Comment (gcapurso) [reply app/templates/administrative-units/administrative-unit/sites/index.hbs:35]: I don't know, in the design it has been changed in Neen. Should I put 'Nee'?

Comment (Windvis) [reply app/templates/administrative-units/administrative-unit/sites/index.hbs:35]: "Neen" is fine as well (although I think it sounds a bit weird), but then it shouldn't be a capital N :smile:

**Review by Windvis — APPROVED** [review]

**Pull request merged**